distribution.pm

```diff
@@ -132,13 +132,14 @@ sub script_run {
         {
             timeout => $bmwqemu::default_timeout,
             output  => '',
-            quiet   => undef
+            quiet   => undef,
+            type_speed => testapi::get_var('VNC_TYPING_LIMIT')
```

Comment:
Update the documentation accordingly

Reply:
As soon as it is complete, when I now update the doc and need to implement changes this gets messy